tests/chainer_tests/functions_tests/pooling_tests/test_roi_max_pooling_2d.py

```diff
@@ -91,9 +91,13 @@ def test_forward_cpu_gpu_equal(self):
 
     def check_backward(self, x_data, roi_data, roi_index_data, y_grad):
         def f(x, rois, roi_indices):
-            return functions.roi_max_pooling_2d(
+            y = functions.roi_max_pooling_2d(
                 x, rois, roi_indices, outsize=self.outsize,
                 spatial_scale=self.spatial_scale)
+            xp = cuda.get_array_module(y)
```

Comment:
It would be nice to add a comment like `# replace -inf with zero for gradient_check`.

Reply:
I update to add comment